client/blocks/comments/post-comment-list.jsx

```diff
@@ -328,6 +387,23 @@ class PostCommentList extends Component {
 			return null;
 		}
 
+		if ( this.props.expandableView && ! this.state.isExpanded ) {
+			// Note this gets the most recent comment/reply of any nested level and shows it.
+			// However, the expanded view renders based on the most recent top level comments. That
+			// means when we expand from only seeing the most recent found here, it may not be
+			// rendered in the longer list if it is a reply but not a reply to one of the most
+			// recent top level comments. Do we find the top level comment corresponding to this
+			// most recent and push it into displayedComments? Do we want to redo how
```

Comment:
I'm not sure if this is exactly the same question that you raised in slack, but I think we should definitely display the whole comment chain if we can for context, otherwise the comment may not make sense or even have the opposite meaning.

Reply:
Yes this is noting the same thing from the conversation in slack. 😁 

>but I think we should definitely display all of the ancestor comments if we can for context, otherwise the comment may not make sense or even have the opposite meaning.

Talking about the expanded view, its not necessarily about showing all ancestor comments for context (its doing that already, rendering the full tree for those comments). But its doing so for the comment trees based off the recentness of the top-level comments and not the replies, which might not necessarily contain the most recent reply we show in the collapsed view (if it was replying on an older top-level comment). 

Combining this with what you noted above about the collapsed view feeling weird, there are a lot of options for what we can do here and what we actually want to show in each view. I imagine @davemart-in will have some input when he gets back as well so I probably wont dive too deep in changing these yet.